README.md

```diff
@@ -79,6 +88,13 @@ Supported ABIs are ilp32 (32-bit soft-float), ilp32d (32-bit hard-float),
 ilp32f (32-bit with single-precision in registers and double in memory, niche
 use only), lp64 lp64f lp64d (same but with 64-bit long and pointers).
 
+
+##ADVICE
+
+You can get an opcode error in the Ubuntu distribution of Linux. 
+This may be because your gcc version is less than 7.3. Upgrade your gcc version to 7.3 and later. 
+With Debian 9.9 deployment you can complete transactions without any problems.
+
```

Comment:
I use gcc-5.4 with Ubuntu 16.04 and gcc-7.4 with Ubuntu 18.04 and haven't seen any problem.  In order for this to be useful, it needs more info about the exact Ubuntu problem you ran into, which probably requires the Ubuntu release version and the exact gcc version used.